README.md

```diff
@@ -25,6 +29,14 @@ Open the HTML manual in a web brower for visual verification
 
     firefox build/manual/build/html/index.html
 
+Convenient editing of definitions is available in nyaml format. For this, definitions need to be converted first from xml to yaml format by the command
+
+    make nyaml
+
+After editing the definitions in nyaml format in the nyaml subdirectories, the following command can be used to update the definitions in nxdl.xml format:
+
+    make nxdl
+
```

Comment:
Add warning that these targets will overwrite files so care should be taken to avoid updating the repo between conversions...

Reply:
Done. Please check if you're happy with my wording